_test/test_sqw_file/test_data_block.m

```diff
@@ -272,7 +272,7 @@ function test_put_get_sqw_block(obj)
             tob = obj.sqw_obj_for_tests;
             dp = dp.put_sqw_block(fid,tob);
 
-            tob.experiment_info.instruments = [];
+            tob.experiment_info.instruments = IX_null_inst();
```

Comment:
Do not think it is necessary here. The test recovers some instrument/sample from hdd. I am not sure what instrument/sample is there -- may be a nul sample so this change does not make a sence.